src/app.js

```diff
@@ -28,7 +28,7 @@ app.use('/api/finances', financeRouter);
 
 
 app.get('/', (req, res) => {
-  res.send('Hello, from Lucas deployment!');
+  res.send('Hello, world!');
```

Comment:
Should we take this out now since we are close to production or no?

Reply:
yes.  it makes us look like noobs.